src/main/webapp/Portal.html

```diff
@@ -134,7 +134,8 @@
 			head.load(cdnEndpoint + "generated/react-bootstrap.min.js");
 			head.load(cdnEndpoint + "generated/sanitize-html.min.js");
 			head.load(cdnEndpoint + "generated/synapse-react-client.production.min.js");
-			head.load(cdnEndpoint + "generated/synapse-react-client.production.styles.css");
+			// SRC scss should be compiled in SWC project
+			// head.load(cdnEndpoint + "generated/synapse-react-client.production.styles.css");
```

Comment:
swc.scss file contains the following code to pull in and build the SRC scss inside the SWC project:

// Load the SCSS build after the minified build so styles with overrides take higher priority
@use 'node_modules/synapse-react-client/dist/style/main.scss' with (
  $primary-action-color: swc.$synapse-blue,
  $secondary-action-color: swc.$synapse-green,
  $bootstrap-4-danger-bg: swc.$synapse-red-error,
  $bootstrap-4-danger-text-color: white
);